.travis.yml

```diff
@@ -95,13 +89,27 @@ install:
 
   # Update the package list and install the Cloud SDK
   - sudo apt-get -qq update
-  - sudo apt-get install google-cloud-sdk -y
+  - sudo apt-get -qq install google-cloud-sdk -y
+
+  # Installing Azure command-line client here as used both in cleaning and cron_job scripts
```

Comment:
The installation of the `azure-cli` has been moved here within the .`travis.yml` file for avoiding duplicates line of code within the `cron_script.sh` and `clean_az_dupl.sh` file. In fact, since they will not always run at the same time, both script needs the azure client to be installed to perform some API queries.